### PR TITLE
sci-libs/opencascade-7.{6.3,7.0} - build against vtk 9.2.2

### PR DESCRIPTION
**Pull request by waebbl**
Closes: https://bugs.gentoo.org/886343
Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

### CODE REVIEW

**Review by ceamac — CHANGES_REQUESTED** [review]
If you choose to backport the fix to opencascade-7.6.3 you need to revbump because you changed a RDEPEND.

Also, for 7.7.0:
- please add `x11-libs/libX11` to RDEPEND
- please check if `x11-libs/libXmu` is really needed

**Comment by waebbl** [comment]
Revbumped both versions and added the libX11 / libXmu patch to both versions.

File: sci-libs/opencascade/opencascade-7.7.0-r1.ebuild [hunk 1]
```diff
@@ -132,24 +133,16 @@ src_configure() {
...
-				-D3RDPARTY_VTK_LIBRARY_DIR="${ESYSROOT}"/usr/$(get_libdir)
-			)
+			mycmakeargs+=( -D3RDPARTY_VTK_INCLUDE_DIR="${ESYSROOT}"/usr/include/vtk-9.2 )
 		elif has_version ">=sci-libs/vtk-9.1.0"; then
-			mycmakeargs+=(
-				-D3RDPARTY_VTK_DIR="${ESYSROOT}"/usr
-				-D3RDPARTY_VTK_INCLUDE_DIR="${ESYSROOT}"/usr/include/vtk-9.1
-				-D3RDPARTY_VTK_LIBRARY_DIR="${ESYSROOT}"/usr/$(get_libdir)
-			)
+			mycmakeargs+=( -D3RDPARTY_VTK_INCLUDE_DIR="${ESYSROOT}"/usr/include/vtk-9.1 )
```
Comment (thesamesam) [inline sci-libs/opencascade/opencascade-7.7.0-r1.ebuild:143]: We should really just use $(ver_cut) here I think and then we can drop a lot of this logic?

Comment (waebbl) [reply sci-libs/opencascade/opencascade-7.7.0-r1.ebuild:143]: Good idea, I'm gonna try it.

Comment (thesamesam) [reply sci-libs/opencascade/opencascade-7.7.0-r1.ebuild:143]: Thanks!

**Pull request closed without merging**